gcn_prot/data/data_input.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 18 09:40:35 2020
+
+@author: Bjorn
+"""
+import torch as t
+import torch.nn.functional as f
+
+
+def VCAInput(n_nodes, n_coord, n_feat, mask=None):
+   
+   '''
+    Method returns Pytorch placeholders for graph convolutional networks.
```

Comment:
Yes, in TensorFlow we would need placeholders. However, what we need for Pytorch is a transformation from the real ( `[v, c, m, y]` output of `__get_index__` at `gcn_prot/data/protein_graph.py`) to the output matrices (features and adjancency) + labels (which is `y`, untouched).